content/03.methods.md

```diff
@@ -1,9 +1,33 @@
-#Materials and Methods
+# Materials and Methods
+## Hetionet
+Hetionet [@doi:10.7554/eLife.26726] is a large heterogenous network that contains pharmacological and biological information.
+This network depicts information in the form of nodes and edges of different types: nodes that represent biological and pharmacological entities and edges which represent relationships each entity may share with itself or another. 
+Currently, Hetionet contains 47,031 nodes with 11 different data types and 2,250,197 edges that represent 24 different relationship types (Figure 1).
+All of the data in this network comes from publicly available databases, such as GWAS Catalog [@doi:10.1093/nar/gkw1133], DrugBank [@doi:10.1093/nar/gkw1133] and more.
+For this project, we focus on the following relationship types: disease associates with a gene (DaG), compounds binding to a gene (CbG), gene interacts with gene (GiG) and compound treating a disease (CtD).
 
-##Dataset
-Talk about dataset - Pubtator
-Talk about preprocessing Pubtator
-Talk about hand annotations for each realtion
+![Hetionet_metagraph](images/figures/metagraph.png)
```

Comment:
I think you should use a figure that shows only the relationship types that you actually used. _OR_ a figure like this one, where all of the types you didn't use are "faded" with, say, 70% transparency, into the background.